proxy/ControlMatcher.cc

```diff
@@ -651,23 +651,21 @@ template <class Data, class MatchResult>
 void
 IpMatcher<Data, MatchResult>::Match(sockaddr const *addr, RequestData *rdata, MatchResult *result) const
 {
-  void *raw;
-  if (ip_map.contains(addr, &raw)) {
-    Data *cur = static_cast<Data *>(raw);
-    ink_assert(cur != nullptr);
-    cur->UpdateMatch(result, rdata);
+  if (auto &&[range, data]{*ip_addrs.find(swoc::IPAddr(addr))}; !range.empty()) {
+    ink_assert(data != nullptr);
+    data->UpdateMatch(result, rdata);
   }
 }
 
 template <class Data, class MatchResult>
 void
 IpMatcher<Data, MatchResult>::Print() const
 {
-  printf("\tIp Matcher with %d elements, %zu ranges.\n", num_el, ip_map.count());
-  for (auto &spot : ip_map) {
-    char b1[INET6_ADDRSTRLEN], b2[INET6_ADDRSTRLEN];
-    printf("\tRange %s - %s ", ats_ip_ntop(spot.min(), b1, sizeof b1), ats_ip_ntop(spot.max(), b2, sizeof b2));
-    static_cast<Data *>(spot.data())->Print();
+  std::string tmp;
+  printf("\tIp Matcher with %d elements, %zu ranges.\n", num_el, ip_addrs.count());
```

Comment:
This looks like debugging remnant?

Reply:
It's part of the `Print` method which is used from a higher layer.